services/webhook/general_test.go

```diff
@@ -64,9 +64,17 @@ func forkTestPayload() *api.ForkPayload {
 }
 
 func pushTestPayload() *api.PushPayload {
+	return pushTestPayloadWithCommitMessage("commit message")
+}
+
+func pushTestPayloadMultilineCommitMessage() *api.PushPayload {
+	return pushTestPayloadWithCommitMessage("This is a commit summary that is longer than 50 characters\n\nThis is the message body.")
+}
+
+func pushTestPayloadWithCommitMessage(message string) *api.PushPayload {
```

Comment:
Wanted to avoid test code duplication here by adding these 2 functions. I'm open for alternative ways to writing my test case.